content/news/2020-11-13-release-plans-and-call-for-contrib.md

```diff
@@ -0,0 +1,48 @@
+type = "news"
+title = "R1 Status Update and Call for Contributors"
+author = "bitigchi"
+date = "2020-11-13 13:00:00+03:00"
+tags = ["r1"]
+
+After nearly 20 years of development, Haiku is very close to its first release. We need your help to make it happen. Here's a status update for R1, and latest project news.
```

Comment:
I wouldn't qualify more than 600 tickets as "very close"

Reply:
IIRC, close to 1000 tickets have been resolved between beta1 and beta2, in a time period of 1.5~ years. Comparing to that, it sounds doable and close enough (not very close, though, I'll edit the "very" part).

Reply:
Active tickets in early 2017: about 2000 for R1 + 1000 for Unscheduled
Active tickets in early 2019: about 1300 for R1 + 1500 for Unscheduled

The total is +200 tickets over two years. But there has been efforts to triage things and move some to "unscheduled". I would'nt say we have solved 1000 tickets between beta1 and beta2. We have made a lot of triaging effort and the remaining tickets are things that actually need work.

Source: http://web.archive.org/web/20190426233529/https://dev.haiku-os.org/roadmap vs http://web.archive.org/web/20170201165934/https://dev.haiku-os.org/roadmap

Reply:
I see. I [checked](https://dev.haiku-os.org/query?status=closed&milestone=%5ER1%2Fbeta2&col=id&col=summary&col=status&col=milestone&col=type&col=priority&col=component&order=priority) the beta2 milestone for this, but triaging etc. sounds reasonable.

Reply:
There's also the fact that people keep opening new tickets, so sometimes it feels like we're going backwards on this. Unfortunately Trac doesn't really provide us with good metrics. But now with beta3 we're trying to use the milestones more accurately and we should get a better idea going forward.

In any case, it's a lot of work still to be done, and the ticket number doesn't necessarily reflect this (there are a lot of regressions that get fixed very quickly, for example, but the old, big, complex tickets remain and accumulate).

And well, "we're never been this close" always applies :)
I guess we have reached the point where "90% of the work is done, now let's start on the remaining 90%" or something. All the boring work that no one wants to do gets postponed to the end, and at some point someone has to dig into it.